admin/src/api/post/content-types/post/lifecycles.js

```diff
@@ -146,7 +146,12 @@ function generateRandomNumber() {
   return Math.floor(Math.random() * 9000000000) + 1000000000;
 }
 
-function validateFields(result) {
+function validateFields(result, is_from_update) {
+  // no need to validate if it's only going to publish or unpublish
+  if (Object.keys(result).length <= 3) {
```

Comment:
What it means? When going to publish, it will have 3 fields only?

Reply:
We should not depend on length of fields. You can check it by fields also, like `is_published` or any other.

Reply:
no one field is good to go.
Publish: 
{
  updatedBy: 1,
  publishedAt: 2024-06-27T09:40:11.092Z,
  updatedAt: 2024-06-27T09:40:11.099Z
}

Unpublish: 
{
  updatedBy: 1,
  publishedAt: null,
  updatedAt: 2024-06-27T09:40:15.483Z
}

Reply:
```
{
  id: 52,
  title: 'reethtehter dgbrdb  fbdgvgbfdhe',
  content: '<html><head></head><body><p>etthth rtjhrthrth</p></body></html>',
  slug: 'post-1',
  published_on: '2024-06-27T09:16:54.467Z',
  is_featured: false,
  summary: 'srgrgr ryht hrtj ryhry',
  blog_content: '<p>etthth rtjhrthrth</p>',
  meta_description: 'rgrhth yryjyjr',
  toc: '<ul style="list-style-type: disc"></ul></li>',
  tags: [
    {
      id: 113,
      name: 'regdh',
      slug: 'regdh',
      createdAt: '2024-06-27T09:16:54.460Z',
      updatedAt: '2024-06-27T09:16:54.460Z'
    }
  ],
  is_published: true,
  keywords: null,
  new_content: null,
  new_toc: null,
  new_blog_content: null,
  is_resource: true,
  reading_time: 1,
  author: { disconnect: [], connect: [ [Object] ] },
  comments: { disconnect: [], connect: [] },
  category: { disconnect: [], connect: [] },
  image: null,
  cta: { disconnect: [], connect: [] },
  updatedBy: 1,
  updatedAt: 2024-06-27T09:42:40.837Z
}
```